src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessStartInfoTests.cs

```diff
@@ -186,5 +188,59 @@ public void TestUseShellExecuteProperty()
             // Calling the getter
             Assert.False(psi.UseShellExecute, "UseShellExecute=true is not supported on onecore.");
         }
+
+        [Fact]
+        public void TestArgumentsProperty()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo();
+            Assert.Equal(string.Empty, psi.Arguments);
+
+            psi = new ProcessStartInfo("filename", "-arg1 -arg2");
+            Assert.Equal("-arg1 -arg2", psi.Arguments);
+        }
+
+        [Fact]
+        public void TestCreateNoWindowProperty()
+        {
+            Process testProcess = CreateProcessInfinite();
+            try
+            {
+                testProcess.StartInfo.CreateNoWindow = true;
+                testProcess.Start();
+
+                Assert.True(testProcess.StartInfo.CreateNoWindow);
+            }
+            finally
+            {
+                if (!testProcess.HasExited)
+                    testProcess.Kill();
+
+                Assert.True(testProcess.WaitForExit(WaitInMS));
+            }
+        }
+
+        [Fact, PlatformSpecific(PlatformID.Windows)]
+        public void TestUserCredentialsPropertiesOnWindows()
+        {
+            // test the defaults here.
+            Assert.Equal(string.Empty, _process.StartInfo.Domain);
+            Assert.Equal(string.Empty, _process.StartInfo.UserName);
+            Assert.Equal(default(SecureString), _process.StartInfo.Password);
+            Assert.False(_process.StartInfo.LoadUserProfile);
```

Comment:
As with other properties, did you want to test that you can roundtrip through these, setting them and then getting back the same values you set?  I assume we're not testing the actual act of launching a process with credentials because we don't have any we can use, and at least in inner loop testing we don't want to add accounts to the machine?